### DISCUSSION TITLE
DERP_VERIFY_CLIENT_URL

### DISCUSSION THREAD

**Discussion by pantigon** [fredliang44/derper-docker#17] [Q&A]
what url do i need to enter when i use tailscale but not headscale?
DERP_VERIFY_CLIENT_URL="" ?

**Comment by europacafe** [comment]
+1
Could it be: DERP_VERIFY_CLIENT_URL="https://headscale.mydomain.com/verify" ?
If the url is my headscale domain.

**Reply by europacafe** [reply]
Please refer to this issue thread after clarification from fredliang, with some screenshot examples of mine
https://github.com/fredliang44/derper-docker/issues/25#issuecomment-3251635496
